### PR TITLE
Migration from Ostrich to Metrics

### PR DESCRIPTION
**Pull request by AnishShah**
So, There is two ways of using StatsReceier : 

1. Use DefaultStatsReceiver directly.
2. Pass StatsReceiver while initializing

I have used the first one which I think is better. What are your thoughts? @szymonm @pankajgupta @caniszczyk 

Now, you can use the stats at `localhost:9990/admin/metrics.json?filtered=true` after passing `-com.twitter.finagle.stats.statsFilter="jvm.*,.*\.p90"`

And please tell me what more changes are needed?

### CODE REVIEW

**Comment by caniszczyk** [comment]
LGTM thanks

File: cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala [hunk 1]
```diff
@@ -29,8 +29,8 @@ case class SuppliedNodeInfo(val node: Node, val initialIterationWeight: Double)
 /**
  * Iteratively analyze links in a bipartite graph.
  * @param graphUtils GraphUtils encapsulating the underlying graph
- * @param resetProbLeft reset probability when iterating from left to right
- * @param resetProbRight reset probability when iterating from right to left
+ * @param resetProbOnLeft reset probability when iterating from left to right
```
Comment (pankajgupta) [inline cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala]: thanks for fixing these names as well

File: cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@ package com.twitter.cassovary.graph.bipartite
 import com.twitter.cassovary.util.SmallBoundedPriorityQueue
 import com.twitter.cassovary.graph.{GraphUtils, Node}
 import com.twitter.logging.Logger
-import com.twitter.ostrich.stats.Stats
+import com.twitter.finagle.stats.DefaultStatsReceiver
```
Comment (pankajgupta) [inline cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala]: here and everywhere else, please maintain alpha ordering of the imports

**Comment by pankajgupta** [comment]
Sorry for the late comments, @AnishShah -- please feel free to have a separate PR for these.

File: cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala [hunk 3]
```diff
@@ -38,6 +38,7 @@ class IterativeLinkAnalyzer(graphUtils: GraphUtils, resetProbOnLeft: Double,
 
   private val graph = graphUtils.graph
   private val log = Logger.get
+  private val statsReceier = DefaultStatsReceiver
```
Comment (pankajgupta) [inline cassovary-core/src/main/scala/com/twitter/cassovary/graph/bipartite/IterativeLinkAnalyzer.scala]: 'statsReceier' is missing a 'v' in its spelling.

File: cassovary-server/src/main/scala/CassovaryServer.scala [hunk 4]
```diff
@@ -60,19 +57,11 @@ object CassovaryServer {
...
-    // start finagle server
-    val server = ServerBuilder()
-          .codec(http.Http())
-          .bindTo(new InetSocketAddress(8888))
-          .name("CassovaryServer")
-          .reportTo(new OstrichStatsReceiver)
-          .build(service)
+    // start Twitter Server
+    val server = Http.serve(":8888", service)
+    onExit{
```
Comment (pankajgupta) [inline cassovary-server/src/main/scala/CassovaryServer.scala]: space before {

File: project/Build.scala [hunk 5]
```diff
@@ -24,10 +24,11 @@ object Cassovary extends Build {
       fastUtilsDependency % "provided",
       "org.mockito" % "mockito-all" % "1.8.5" % "test",
       util("core"),
-      "com.twitter" %% "ostrich" % "9.1.0",
       util("logging"),
       "org.scalatest" %% "scalatest" % "1.9.2" % "test",
-      "junit" % "junit" % "4.10" % "test"
+      "junit" % "junit" % "4.10" % "test",
+      "com.twitter" %% "finagle-stats" % "6.20.0",
+      "com.twitter" %% "twitter-server" % "1.9.0"
```
Comment (pankajgupta) [inline project/Build.scala]: is twitter-server a shared setting or only for cassovary-server?

File: project/Build.scala [hunk 6]
```diff
@@ -24,10 +24,11 @@ object Cassovary extends Build {
       fastUtilsDependency % "provided",
       "org.mockito" % "mockito-all" % "1.8.5" % "test",
       util("core"),
-      "com.twitter" %% "ostrich" % "9.1.0",
       util("logging"),
       "org.scalatest" %% "scalatest" % "1.9.2" % "test",
-      "junit" % "junit" % "4.10" % "test"
+      "junit" % "junit" % "4.10" % "test",
+      "com.twitter" %% "finagle-stats" % "6.20.0",
```
Comment (pankajgupta) [inline project/Build.scala]: looks like you can move to 6.24.0

**Comment by pankajgupta** [comment]
Finally, please modify cassovary-server/ServerREADME.md as well.

**Comment by AnishShah** [comment]
Cool :+1: I'll do it in sometime..

**Pull request merged**